app/containers/ItunesContainer/tests/reducer.test.js

```diff
@@ -44,4 +46,52 @@ describe('ItunesContainer reducer tests', () => {
       })
     ).toEqual(expectedResult);
   });
+  it('should ensure when CLEAR_ARTIST is dispatched , it clears the trackDetails', () => {
```

Comment:
Maybe phrase it better `it(should ensure that trackDetails are cleared when CLEAR_ARTIST is dispatched)` Think of it as a reserved keyword and don't use it again.